orbit/pkg/update/notifications.go

```diff
@@ -501,14 +501,14 @@ func (w *windowsMDMBitlockerConfigFetcher) attemptBitlockerEncryption(notifs fle
 		ClientError:   bitlockerError,
 	}
 
+	err = w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
-		log.Error().Err(err).Msg("failed to encrypt the volume")
+		log.Error().Err(err).Msg("failed to send encryption result to Fleet Server")
 		return
 	}
 
-	err = w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
```

Comment:
Looks like there's some function call missing before this `if err != nil`? The same error value is checked twice (line 505 and line 510) without anything else assigning to `err` in between.